docs/sources/writing-guide/references/index.md

```diff
@@ -122,7 +122,7 @@ Hugo references addressed across different products' docs, such as from `/docs/g
 
 To avoid broken links in these situations on grafana.com, use regular Markdown link syntax (`[link text](/docs/repo/version/folder/file/)`) instead of Hugo references. To ensure the links work in local builds, staging environments, and the live website, you **shouldn't** use a fully qualified URL with `https://grafana.com` for links to other content on grafana.com.
```

Comment:
I think "in these situations" is a bit unclear because the preceding text involves two distinct scenarios--a) addressing other versions, b) cross-product links. Also, I suggest defining "fully qualified URL".

Reply:
I think this is a very fair point but I don't want to block this PR that fixes incorrect syntax on unrelated changes.

If you have some time to submit some changes in a separate PR, I would happily review those changes.

Reply:
In the mean time, would you mind removing the request for changes so that this PR can be unblocked?

Reply:
Thank you for the suggestions. It would be great if you'd be willing to submit a new PR. This was just a quick syntax fix, but you ahve some good points.